using-valtimo/access-control/auto-deployment-access-control.md

```diff
@@ -53,7 +53,7 @@ The example below defines 2 permissions:
                     "path": "$.height",
                     "operator": "<",
                     "value": 20000,
-                    "clazz": "int"
```

Comment:
There is another "int" in https://docs.valtimo.nl/reference/modules/authorization#expression

Reply:
Good catch, fixed it